.test-infra/jenkins/job_PostCommit_Python_MongoDBIO_Load_Test.groovy

```diff
@@ -51,7 +51,7 @@ job(jobName) {
       rootBuildScriptDir(common.checkoutDir)
       common.setGradleSwitches(delegate)
       switches("-Popts=\'${common.mapToArgString(pipelineOptions)}\'")
-      tasks(":sdks:python:test-suites:dataflow:py35:mongodbioIT")
+      tasks(":sdks:python:test-suites:dataflow:mongodbioIT")
```

Comment:
Note that this is a load test, and we also have integration test suite: https://github.com/apache/beam/blob/d971ba13b8ed31de3413558f701d752cbf202d47/.test-infra/jenkins/job_PostCommit_Python_MongoDBIO_IT.groovy#L35